pom.xml

```diff
@@ -823,14 +823,4 @@
       <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
     </repository>
   </distributionManagement>
-  <repositories>
-    <repository>
-      <snapshots>
-        <enabled>false</enabled>
-      </snapshots>
-      <id>bintray-mockito-maven</id>
-      <name>bintray</name>
-      <url>https://dl.bintray.com/mockito/maven</url>
-    </repository>
-  </repositories>
```

Comment:
Please revert this change. I would like to be able to use the latest Mockito, and for some reason, they don't publish their latest stuff to Central.

Reply:
Well, this is quite sad situation. Maven is unable to download this dependency with my current settings.xml :(

Reply:
Ok, I'll revert it and try to find the root cause in my local settings.

Reply:
Resolved. I had a `<mirror>` in my `settings.xml` with `<mirrorOf>*</mirrorOf>` ... had to change it to `<mirrorOf>!bintray-mockito-maven,*</mirrorOf>`